game/views.py

```
@@ -2550,3 +2554,40 @@ def remove_featured_badge(request, badge_id):
+            as_attachment=True,
+            filename=f"{safe_filename}.png"
+        )
+    except (
+        FileNotFoundError,
+        OSError,
+    ) as exc:
+        logger.error(
+            "Badge generation failed for achievement %s: %s",
+            achievement_id,
+            exc
+        )
```

Ruff (0.15.15)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)